### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by tiffanychu90**

### CODE REVIEW

File: dags/public-health/COVID-19_Dashboard_README.md [hunk 1]
```diff
@@ -9,14 +9,18 @@ The Johns Hopkins Center for Systems Science and Engineering has open sourced da
...
 
-**3/13/2020 update:** JHU's CSVs will be at the state level, and not at the city/county level anymore, [as noted in their GitHub issue](https://github.com/CSSEGISandData/COVID-19/issues/382). Since JHU's feature layers weren't connecting to our dashboard, we adapted our ETL to continue to grab province/state level data for the world and will publish these as 2 public ESRI feature layers (#1, #2).
 
-In addition, we are scraping the websites for Southern California counties belonging in the Southern California Association of Governments (SCAG) region. We have data from Los Angeles, Orange County, and Imperial Counties. In the coming days, we will add Ventura, Riverside, and San Bernardino Counties. We have combined JHU county data up to 3/12/2020 with our own compilation of counts, and will publish these as 2 public ESRI feature layers (#3, #4).
+**3/16/2020 update:** We caught our ETL not working as expected over the weekend of 3/14-3/15. We manually inputted case counts for 3/13/2020 for Los Angeles County (53), Orange County (13), and Imperial County (0). We are not able to fill in the other columns for 3/13/2020.
```
[the remark this answers is not in the document]
Comment (tiffanychu90) [reply dags/public-health/COVID-19_Dashboard_README.md]: I'll take this section out for now.

**Comment by ian-r-rose** [comment]
This has now duplicated the `README`, since @hunterowens moved it in #153

**Comment by tiffanychu90** [comment]
I know, I'm removing it the duplicate, but the pre-commit checks are taking forever.

**Comment by ian-r-rose** [comment]
Yeah, the first time you run it, it installs everything and takes a long time. After that it should be more reasonable.

**Pull request merged**